docs/userguide/modules/creating-service-bindings/pages/binding-options.adoc

```diff
@@ -182,4 +182,59 @@ spec:
 ----
 <1>  Database service
 <2> Optional "id" field
-<3>  Translation service
\ No newline at end of file
+<3>  Translation service
+
+== Binding workloads from a label selector
+
+Sometimes, it may be useful to specify the workload being bound by using a label selector.  For
+instance, you may want to bind a service to every workload with the label `environment: production`
+set.  SBO is able to support this kind of binding by being able to bind with label selectors.  As an
+example, to project a single secret into multiple workloads, set up your application field like so:
+[source,yaml]
+----
+apiVersion: binding.operators.coreos.com/v1alpha1
+kind: ServiceBinding
+metadata:
+  name: multi-application-binding
+  namespace: service-binding-demo
+spec:
+  application:
+    labelSelector:
+      matchLabels:
+        environment: production
+    group: apps
+    version: v1
+    resource: deployments
+  services:
+    group: ""
+    version: v1
+    kind: Secret
+    name: super-secret-data
+----
+
+Or, using the spec API:
```

Comment:
Replace L215 with this:
.Example of `ServiceBinding` CR in the `servicebinding.io` API: